rest-server/src/components/auth/authController.js

```diff
@@ -0,0 +1,33 @@
+import User from '../user/userController';
+import bcrypt from 'bcrypt';
+import jwt from 'jsonwebtoken';
+import { error } from '../../lib/log';
+
+export default class Auth {
+  static async createTokenForUser(userEmail) {
```

Comment:
should we make token creation and other authentication stuff a part of middleware?